artifactory/services/utils/artifactoryutils.go

```diff
@@ -328,7 +330,7 @@ func loadMissingProperties(reader *content.ContentReader, readerWithProps *conte
 // buffer - Search result buffer (sorted) Key -> relative path, value -> ResultItem.
 // crWithProps - File containing all the results with proprties.
 // resultWriter - Sorted search result with props.
-func updateProps(crWithProps *content.ContentReader, resultWriter *content.ContentWriter, buffer map[string]*ResultItem) error {
+func updateProps(crWithProps *content.ContentReader, resultWriter *content.ContentWriter, buffer map[string]*ResultItem, writeOrder []*ResultItem) error {
```

Comment:
The function description describes all the arguments, except for the new one. The description is also not accurate anymore ("...sort the buffered keys...").